lib/models/ventilation_guide.dart

```diff
@@ -0,0 +1,20 @@
+class VentilationGuideItem {
```

Comment:
:+1: :+1:  for keeping with modelling the data @ericjiang97

Reply:
:)